service/Checkout.php

```diff
@@ -24,10 +24,56 @@
 
 namespace Adyen\PrestaShop\service;
 
+use Adyen\PrestaShop\service\adapter\classes\ServiceLocator;
+use Cart;
+
 class Checkout extends \Adyen\Service\Checkout
 {
+    const PAYMENT_METHOD_STEP = 'checkout-payment-step';
```

Comment:
do we have the same steps for one step checkout as well?

Reply:
On 1.7 the checkout is always considered as [one-page](https://www.prestashop.com/forums/topic/628709-enable-standard-5-step-checkout/) since technically you stay on the same controller while moving trough the steps. So the steps and the strings representing them will always be the same